setup.bat

```diff
@@ -0,0 +1,21 @@
+@ECHO OFF
+ECHO Gathering the force
+MKDIR  .venv
+ECHO CREATING .env file
+ECHO DEVELOPMENT=1 >> .env
+ECHO SECRET_KEY="your_secret_key_here" >> .env
+ECHO SITE_NAME="localhost" >> .env
+ECHO Installing pipenv for development use
+pip install pipenv
+CLS
+@ECHO OFF
+ECHO Installing requirements.txt
+pipenv install -r requirements.txt
+
+ECHO ============================
+ECHO Please edit .env file and add your secret key
```

Comment:
Could you add something for Windows like you have in the bash script to generate a random `SECRET_KEY`?

Reply:
I can try add it with a small function.  for bash we have built in openssl.
But will add something

Reply:
Sure. Was just a suggestion.